index.html

```diff
@@ -462,11 +462,244 @@ <h3>Data Model</h3>
         </p>
 
-        <div class="issue">Add examples of common Controller documents, such as
-controller documents published on a ledger-based registry, or on a mutable medium in
-combination with an integrity protection mechanism such as Hashlinks.
+        <div class="issue">
+Add examples of common Controller documents.
         </div>
 
+        <p class="note"
+           title="Property names used in map of different types">
+The property names `id`, `type`, and `controller` can be present in map of
+different types with possible differences in constraints.
+        </p>
+
+        <section>
+          <h2>Controller Documents</h2>
+
+          <p>
+The following sections define the properties in a [=controller document=],
+including whether these properties are required or optional. These properties
+describe relationships between the [=subject=] and the value of the
+property.
+          </p>
+
+          <p>
+The following tables contain informative references for the core properties
+defined by this specification, with expected values, and whether or not they are
+required. The property names in the tables are linked to the normative
+definitions and more detailed descriptions of each property.
+          </p>
+
+          <table class="simple">
+            <thead>
+              <tr>
+                <th>Property</th>
+                <th>Required?</th>
+                <th>Value constraints</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>`id`</td>
+                <td>yes</td>
+                <td>
+A <a data-cite="INFRA#string">string</a> that conforms to the URL syntax
+defined in Section [[[#subjects]]].
+                </td>
+              </tr>
+              <tr>
+                <td>`alsoKnownAs`</td>
+                <td>no</td>
+                <td>
+A <a data-cite="INFRA#ordered-set">set</a> of
+<a data-cite="INFRA#string">strings</a> that conform to the rules of
+[[RFC3986]] for URIs as defined in Section [[[#also-known-as]]].
+                </td>
+              </tr>
+              <tr>
+                <td>`controller`</td>
+                <td>no</td>
+                <td>
+A <a data-cite="INFRA#string">string</a> or a
+<a data-cite="INFRA#ordered-set">set</a> of
+<a data-cite="INFRA#string">strings</a> that conform to the URL syntax
+defined in the [[[URL]]] as defined in Section [[[#controllers]]].
+                </td>
+              </tr>
+              <tr>
+                <td>`verificationMethod`</td>
+                <td>no</td>
+                <td>
+A <a data-cite="INFRA#ordered-set">set</a> of
+[=verification method=] <a data-cite="INFRA#ordered-map">maps</a>
+that conform to the rules in Section [[[#verification-methods]]].
+                </td>
+              </tr>
+              <tr>
+                <td>`authentication`</td>
+                <td>no</td>
+                <td rowspan="5">
```

Comment:
If this works, then the following suggestion with the `none` cells can be rejected. It *might* be good to list the properties to which this cell applies, within this cell.
```suggestion
                <td rowspan="5" style="vertical-align: middle;">
```

Reply:
I ended up implementing this instead: 345e8e42f80832a4b8a539f0b3b6ce28d8ded863 ... which builds off of one of your corrections that helped me see a way forward w/o using "rowspan".